application/play/java/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build app using Maven
+FROM mozilla/sbt:latest
+
+RUN curl -sLJO https://deb.secrethub.io/amd64
+RUN dpkg -i secrethub-cli-amd64.deb
+RUN rm secrethub-cli-amd64.deb
+RUN printf 'demo\ndemo\n' | sbt new playframework/play-scala-seed.g8
+# Install SecretHub
+
+COPY application.conf demo/conf/application.conf
+COPY HomeController.java demo/app/controllers/HomeController.java
```

Comment:
If this is the standard project structure for apps build with the play framework, I think it would be better to also have this structure in example code (instead of having `application.conf` and `HomeController.java` next to each other and copying them to their correct locations).

Reply:
This would mean to add 100+ files, which I don't think this is what we want.

Reply:
Yeah in that case I'd say we can leave it like this.